### PR TITLE
Fix Empty Webhook URL When Switching Workflow Trigger Type

### LINKED ISSUE #2050: webhook url does not show in the workflow configuration modal when switching triggers from Cron to Events
Video: https://www.loom.com/share/ce470dc0f2e3496c8a4abc703bf19b96?sid=8f7a917e-0fdc-41ea-9067-f2ead157a67a

### PR DESCRIPTION
**Pull request by elias-ba**
### Description

This PR fixes the empty webhook URL input when one switches a workflow trigger type.

Closes #2050 

### Validation steps

1. Open a workflow
2. Switch it's trigger from CRON to Webhook and vice versa
3. Note that the Webhook URL input is not empty

### Pre-submission checklist

- [x] I have performed a **self-review** of my code.
- [x] I have implemented and tested all related **authorization policies**. (e.g., `:owner`, `:admin`, `:editor`, `:viewer`)
- [x] I have updated the **changelog**.

### CODE REVIEW

File: lib/lightning_web/live/workflow_live/edit.ex [hunk 1]
```diff
@@ -1939,16 +1939,8 @@ defmodule LightningWeb.WorkflowLive.Edit do
...
-    trigger
-    |> case do
-      %{type: :webhook, id: id} ->
-        Routes.webhooks_url(LightningWeb.Endpoint, :create, [id])
-
-      _ ->
-        nil
-    end
-  end
+  defp webhook_url(%{id: id}),
```
Comment (midigofrank) [inline lib/lightning_web/live/workflow_live/edit.ex:1942]: Given that verified routes are currently favoured over route helpers, could you update this to use [url/1](https://hexdocs.pm/phoenix/Phoenix.VerifiedRoutes.html#url/1) instead?

**Review by midigofrank — CHANGES_REQUESTED** [review]
Wait, that is the fix? 😅  I was expecting to see an `if..else` somewhere. 

Could you also write a test for this just to be sure?

File: lib/lightning_web/live/workflow_live/edit.ex (hunk 1 above)
Comment (elias-ba) [reply lib/lightning_web/live/workflow_live/edit.ex]: Done

**Comment by elias-ba** [comment]
Hey @midigofrank thanks for the review. I have handled your points. Would you mind giving me another round of review ? Thanks man

**Review by midigofrank — APPROVED** [review]

**Review by christad92 — APPROVED** [review]
Looks good to me

**Pull request merged**